src/components/order-service/order-service.controller.ts

```diff
@@ -48,20 +64,25 @@ export class OrderServiceController implements angular.IController {
   }
 
   public configureService (plan: any) {
-    // this.ctrl.selectedPlan = plan;
-    this.ctrl.serviceLongDescription = plan;
+    this.ctrl.selectedPlan = plan;
     this.gotoStep(this.ctrl.steps[1]);
   }
 
-  public orderService() {
-    this.gotoStep(this.ctrl.steps[2]);
-  }
-
-  public toggleAdvOps() {
-    let advHref = document.querySelector('#adv-ops-href');
-    let advOps = document.querySelector('#adv-ops');
-    angular.element( advHref ).toggleClass('collapsed');
-    angular.element( advOps ).toggleClass('collapse');
+  public provisionService() {
+    let serviceInstance = this.makeServiceInstance();
+    this.DataService.create({
+      group: 'servicecatalog.k8s.io',
+      resource: 'instances'
+    }, null, serviceInstance, {
+      namespace: this.ctrl.selectedProject.metadata.name
+    }).then(() => {
+      // TODO: Handle async responses
+      this.ctrl.orderComplete = true;
+      this.ctrl.error = null;
+      this.gotoStep(this.ctrl.steps[2]);
```

Comment:
we will probably want named references to steps since the number of steps is going to change depending on the service class, but we can deal with that when we start changing the number of steps.